person.rb

```diff
@@ -0,0 +1,28 @@
+class Person
+  attr_accessor :id, :name, :age
+
+  def initialize(name = 'Unknown', age = nil, parent_permission: true)
```

Comment:
- [ ] Great job so far @mahdinoori2000 There are a few changes I will suggest you implement and you'll be on your way to the next Milestone :wink: 
- [ ] Per the project requirements, you were to create a `Constructor with name, age, and parent_permission as parameters. name and parent_permission are optional and have default values of "Unknown" and true.` :+1: 
- [ ] However currently, you have also given the parameter `age` a default value of `nil`, however this wasn't stated in the project requirements so kindly change that  since `age` needs to be a required parameter :wink: